docs/about.md

```diff
@@ -1,27 +1,27 @@
+MobileCoin has developed a Software Developer Kit (**SDK**) for companies or independent developers
+who want to develop a mobile wallet app. This mobile wallet app will enable users to _send_ and
+_receive_ MobileCoins ([**MOB**](glossary.md)) via their mobile devices. The MobileCoin SDK
+Architecture provides a layer of business logic and cryptographic operations to enable companies and
+developers to get started quickly in developing their wallet app.
 
-For more information about the MobileCoin Network, check out the \[MobileCoin Ecosystem]\(link here?) -- a brief document about the MobileCoin open-source software network.
+For more information about the MobileCoin Network, check out the
+[MobileCoin Developer Portal](https://developers.mobilecoin.com/).
 
-This manual provides a comprehensive overview of the MobileCoin SDK Architecture, and how to utilize the MobileCoin SDK when developing a mobile wallet app, including how the **account key** operates, and how to send and receive transactions.
+This manual provides a comprehensive overview of the MobileCoin SDK Architecture, and how to utilize
+the MobileCoin SDK when developing a mobile wallet app, including how the **account key** operates,
+and how to send and receive transactions.
 
 ## Who should read this guide
 
-This guide should be read and referenced by developers, who have been entrusted with setting up and managing the integration of MobileCoin wallet functionality on a messaging service to enable users to exchange MOB. The developer will be implementing a unique MobileCoin wallet application to integrate with MobileCoin’s network.
+This guide should be read and referenced by developers who have been entrusted with setting up and
+managing the integration of MobileCoin wallet functionality within an Android mobile application.
+The developer will be implementing a unique MobileCoin wallet application to integrate with
+MobileCoin’s network.
 
 ## How to use this guide
 
-Take a moment to familiarize yourself with this iOS/SWIFT Integration Guide by reading the Table of Contents. Each section referenced in the TOC should be read carefully so that you can quickly and easily integrate your wallet app with the MobileCoin network.
-
-### Special symbols used in this guide
-
-At times when we want to emphasize important information and valuable tips associated with the MobileCoin mobile integration app, we'll be using the the **NOTE** and **WARNING** notations with following meanings that you should heed:
-
-{% hint style="info" %}
-Important information concerning the integration of your MobileCoin wallet app with the MobileCoin network.
-{% endhint %}
-
-{% hint style="danger" %}
-Failure to follow directions may result in preventing your MobileCoin wallet app from operating successfully on the MobileCoin network.
-{% endhint %}
+Take a moment to familiarize yourself with this iOS/SWIFT Integration Guide by reading the Table of
```

Comment:
why does it say iOS/Swift ?

Reply:
Good question... Changed this to Android SDK Integration Guide